src/get-bounding-client-rect.js

```diff
@@ -27,15 +27,14 @@ import {
 } from './layout-rect';
 import {isConnectedNode} from './dom';
 
-const nativeClientRect = Element.prototype.getBoundingClientRect;
-
 /**
  * Polyfill for Node.getBoundingClientRect API.
  * @this {!Element}
  * @return {!ClientRect|!LayoutRectDef}
  */
 function getBoundingClientRect() {
   if (isConnectedNode(this)) {
+    const nativeClientRect = Element.prototype.getBoundingClientRect;
```

Comment:
This is incorrect, we're patching `Element.prototype.getBoundingClientRect`. We have to store the value from before the patch.

Reply:
Ah, I see. Apologies for misunderstanding this. That makes sense.

Reply:
It's an easy mistake! Fix is in https://github.com/ampproject/amphtml/pull/22269.